config/dependencies/gateway-api/gateway/gateway.yaml

```diff
@@ -8,12 +8,9 @@ metadata:
 spec:
   gatewayClassName: istio
   listeners:
-  - name: default
+  - name: http
     port: 80
     protocol: HTTP
     allowedRoutes:
       namespaces:
         from: All
-  addresses:
```

Comment:
glad we can remove this,but does it cause any issues with other user-guides?

Reply:
On a sail install this part of the spec gets populated when it creates the loadbalancer service so it should be present anyway. I'll double check that I didn't miss any user guides though.